### ISSUE TITLE
Can't copy to or paste from clipboard in the application on macOS

### ISSUE THREAD

**Issue by LiamHz** [KoltesDigital/shadertoy-exporter#15]

**Comment by KoltesDigital** [comment]
Do you still have the issue? I discussed with some people on macOS and it works fine.

**Comment by LiamHz** [comment]
I still have this issue, but I haven't been using Shadertoy much recently --> I wouldn't prioritize this issue unless other people start running into it as well.

**Comment by munrocket** [comment]
I can install it on macOS, but can't open any shader in application. Shadertoy web page opened but without webgl.

**Comment by KoltesDigital** [comment]
@munsocket obviously this is off-topic, don't be afraid to open a new ticket and to give more info so that I or someone else can help you.

**Comment by bloerwald** [comment]
> Do you still have the issue?

Yes, 1.5.3 still has this issue. Can neither paste nor copy. cmd+a also doesn't select things and cmd+z doesn't undo, so it seems like all those text editing things are bork.

**Comment by runevision** [comment]
I have the copy paste issue too.

It's worth noting that if a Shadertoy URL is in the clipboard when the application is opened, it does load. So that functionality can correctly read from the clipboard. However, any copy, cut, or paste interactions inside the application have no effect. This is both for the sidebar text fields (like URL) and inside the Shadertoy code editor.
